smlb/metrics.py

```diff
@@ -579,6 +579,99 @@ def _evaluate(self, true, pred):
         return np.mean(ContinuousRankedProbabilityScore()._evaluate(true, pred))
 
 
+class StandardConfidence(ScalarEvaluationMetric):
+    '''Fraction of the time that the magnitude of a residual was less than the uncertainty estimate.
+    Evaluates both relationship between uncertainty estimate and the residual, as well as the
+    overal normalization. Does not depend on the predicted value, only the residual
```

Comment:
```suggestion
    """Fraction of the time that the magnitude of the residual is less than the predicted standard deviation.
    Standard confidence evaluates the quality of the predicted uncertainty estimates, both in terms of individual predictions and overall normalization.
    Does not depend on the predicted values, only the residuals.
```
Feel free to iterate.